code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -346,6 +346,7 @@
 	for(var/datum/reagent/R in M.reagents.reagent_list)
 		if(R != src)
 			M.reagents.remove_reagent(R.id,2.5)
+			M.client.dir = NORTH
```

Comment:
Id prefer it just removed it like everything else so you dont need special client presence check